app/controllers/transcribe_controller.rb

```diff
@@ -312,6 +312,37 @@ def still_editing
     render nothing: true
   end
 
+  def goto_next_untranscribed_page
+    
+    next_page_path = landing_page_path
+    flash[:notice] = "There are no more pages to transcribe in this collection! Check out some of our other projects."
+
+    if @work.next_untranscribed_page
+      flash[:notice] = "Here's another page in this work, if you'd like to continue."
+      next_page_path = collection_transcribe_page_path(@work.collection.owner, @work.collection, @work, @work.next_untranscribed_page)
```

Comment:
This one worries me a bit.  When you get to the end, it makes sense to be sent "elsewhere" but when you are inside of a work and the next page is already transcribed...  What is the right behavior?  We don't want to lose them, but it would be jarring to be plopped elsewhere in the work.
You might also want to read/review the next page.
At a minimum we should think about changing the message to make it clearer.  At a maximum, I think we should skip this scenario.

Reply:
I think we want to send them to the next page even though it has been transcribed.

Reply:
This only happens on the last page. So, if a user is mid-work and the next page is already transcribed, they still just go to the next page as expected. The only difference here is that on the last page they get taken to a page in the same work that is incomplete.